webgme/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for webgme v2.6.0 
```

Comment:
Try doing it with only 3 files: `index.d.ts`, `tsconfig.json`, `webgme-tests.d.ts`